### PR TITLE
build: add script to test and booster

### PR DESCRIPTION
**Pull request by danbev**
I've been using this locally to verify things are working correctly.
This script assumes that the nodejs-rest-http is clones into the parent
directory.

### CODE REVIEW

File: make-plus-booster-test.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+booster_image_name=nodejs-rest-http-s2i-rhoar
+
+make
+
+# Build the docker image
+docker build -t $image_name --build-arg NODE_VERSION=$node_version --build-arg NPM_VERSION=$npm_version .
+
+pushd ../nodejs-rest-http/
+
+s2i build . ${image_name}:latest $booster_image_name
```
Comment (helio-frota) [inline make-plus-booster-test.sh]: works fine even without `--pull-policy never` ?

Comment (helio-frota) [reply make-plus-booster-test.sh]: other than that is good to have a script like this :+1:

Comment (helio-frota) [reply make-plus-booster-test.sh]: Tested locally after delete all the images and it is working fine.

Comment (danbev) [reply make-plus-booster-test.sh]: Thanks @helio-frota, I'll add the `--pull-policy` to the script.

**Comment by danbev** [comment]
> Do you think it would live more comfortably in the test directory?

Good point, yeah that would be better. I'll move it. Thanks

**Pull request merged**